src/test/scala/no/ndla/imageapi/service/TagsServiceTest.scala

```diff
@@ -26,48 +26,4 @@ class TagsServiceTest extends UnitSuite with TestEnvironment {
     service.getISO639("http://psi.some.url.org/#ZZZ") should equal (None)
```

Comment:
Ah, se der ja. Bare jeg som ikke kan lese :)
Hovedproblemet ligger kanskje i at `new URL(TopicAPIUrl + nid).openStream` er vanskelig å mocke bort. Om så kunne det kanskje gått an å endre `forContent` til å ta en `InputStream` som parameter. Da kan man sende en egen `InputStream` i testene (http://alvinalexander.com/scala/how-convert-string-to-inputstream-unit-testing).
I tilegg trenger man da en ny "wrapper-funksjon" som igjen kaller funksjonen som mottar en `InputStream`:  `def forImage(nid: String): Try[List[ImageTag]] = forImage(new URL(TopicAPIUrl + nid).openStream)`